### DISCUSSION TITLE
Import css file into MDX docs

### DISCUSSION THREAD

**Discussion by argdesign** [storybookjs/storybook#21756] [Help]
Hi! I’m using MDX to create my component’s documentation, this is how I am adding css code inside my mdx file:

```
<style>
    {`
    .d-flex{
        display:flex;
    `}
</style>
```

wich works fine but I was wondering if is there a way to import css file into my mdx file, for example 

import "../src/stories/styles.scss"; 

I tried to do it but didn’t work, my project is in Angular

**Comment by aldrichdev** [comment]
You can definitely use this syntax in an `.mdx` file with Storybook 7. It works for me:

```
import 'src/my-styles.css';
```

Storybook may not support `.scss` files by default... you may want to research a bit or check this: https://storybook.js.org/recipes/sass

**Comment by chrisandrew-dev** [comment]
In the latest boilerplate, moving rulesets from 'Configure.mdx' to 'style.css', and importing them back into 'Configure.mdx' yields a good first impression. However, at least one class, 'sb-container', is ignored.
